changelog/gc_cleanup.dd

```diff
@@ -0,0 +1,26 @@
+GC cleanup can now be configured as a DRT GC option
+
+The default cleanup method for the GC is to unconditionally run a
+collection before runtime termination to finalize objects
+that are still alive and hold resources that affect system state outside
+the current process. This combines the worst of possible alternatives:
+it can cause a considerable delay and does not guarantee finalization
+of all objects as roots might still exist.
+
+The cleanup behaviour can now be configured by a DRT option to the
+$(LINK2 $(ROOT_DIR)spec/garbage.html, GC configuration),
+e.g. by passing `--DRT-gcopt=cleanup:none` on the command
+line. Three options are provided:
+
+$(UL
+  $(LI collect: run a collection (the default fo backward compatibility))
```

Comment:
```suggestion
  $(LI collect: run a collection (the default for backward compatibility))
```

Reply:
Thanks, amended the change. Is it ok to use githubs "Commit suggestion" or do I have to squash afterwards anyway?

Reply:
Yes it is. Reviewers can use the `auto-merge-squash` option or select it manually in the GitHub merge select menu:

![image](https://user-images.githubusercontent.com/4370550/50699246-a0eb2c00-1047-11e9-8354-1ba0d27f1857.png)

Though of course, it doesn't hurt to squash yourself.